odoo/addons/base/models/assetsbundle.py

```diff
@@ -99,6 +99,12 @@ def subber(match):
     ).strip()
     return result
 
+try:
+    import rjsmin as rjsminlib
+    rjsmin = rjsminlib.jsmin
+except ImportError:
+    rjsmin = regex_jsmin
```

Comment:
```suggestion
try:
    from rjsmin import rjsmin
except ImportError:
    pass
```
and revert the name change on the function. Could also use `contextlib.suppress`, but that might require an additional import.